Failure definitely related:

```
======================================================================
ERROR: corehq.apps.domain.tests.test_domain_calculated_properties:DomainCalculatedPropertiesTest.test_sanity
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/domain/tests/test_domain_calculated_properties.py", line 37, in test_sanity
    all_stats = all_domain_stats()
  File "/vendor/lib/python3.6/site-packages/quickcache/quickcache.py", line 42, in inner
    return helper(*args, **kwargs)
  File "/vendor/lib/python3.6/site-packages/quickcache/quickcache_helper.py", line 179, in __call__
    return self.call(*args, **kwargs)
  File "/vendor/lib/python3.6/site-packages/quickcache/quickcache_helper.py", line 75, in call
    self.cache.set(key, content)
  File "/vendor/lib/python3.6/site-packages/quickcache/cache_helpers.py", line 82, in set
    cache.set(key, value)
  File "/vendor/lib/python3.6/site-packages/quickcache/cache_helpers.py", line 33, in set
    return self.cache.set(self.prefixed_key(key), value, timeout=self.timeout)
  File "/vendor/lib/python3.6/site-packages/django_redis/cache.py", line 32, in _decorator
    return method(self, *args, **kwargs)
  File "/vendor/lib/python3.6/site-packages/django_redis/cache.py", line 67, in set
    return self.client.set(*args, **kwargs)
  File "/vendor/lib/python3.6/site-packages/django_redis/client/default.py", line 115, in set
    nvalue = self.encode(value)
  File "/vendor/lib/python3.6/site-packages/django_redis/client/default.py", line 327, in encode
    value = self._serializer.dumps(value)
  File "/vendor/lib/python3.6/site-packages/django_redis/serializers/pickle.py", line 32, in dumps
    return pickle.dumps(value, self._pickle_version)
AttributeError: Can't pickle local object 'all_domain_stats.<locals>.<lambda>'
```

--- a/corehq/apps/domain/calculations.py
+++ b/corehq/apps/domain/calculations.py
@@ -346,7 +346,7 @@ def all_domain_stats():
     commcare_counts = defaultdict(int)
 
     for row in CouchUser.get_db().view('users/by_domain', startkey=["active"],
-                             endkey=["active", {}], group_level=3).all():
+                                       endkey=["active", {}], group_level=3).all():
         _, domain, doc_type = row['key']
         value = row['value']
         {

--- a/corehq/apps/domain/calculations.py
+++ b/corehq/apps/domain/calculations.py
@@ -342,8 +342,8 @@ def dom_calc(calc_tag, dom, extra_arg=''):
 
 @quickcache([], timeout=60 * 60)
 def all_domain_stats():
-    webuser_counts = defaultdict(lambda: 0)
-    commcare_counts = defaultdict(lambda: 0)
+    webuser_counts = defaultdict(int)
+    commcare_counts = defaultdict(int)
 
     for row in CouchUser.get_db().view('users/by_domain', startkey=["active"],
                              endkey=["active", {}], group_level=3).all():

